### PR TITLE
Add Hyper app for armhf and arm64

### LINKED ISSUE #744: Hyper
[Hyper](https://hyper.is/) is an electron based terminal. It can run on Linux but doe not seem to be able to run on the Raspberry Pi.
 
I think users who use the CLI will find this app useful. 

The Github repo is [here](https://github.com/vercel/hyper/tree/3.0.2)

### PR DESCRIPTION
**Pull request by Jai-JAP**
Closes #744 
Also added auto update scripts that fetches new releases from https://github.com/Jai-JAP/hyper-arm-builds

### CODE REVIEW

File: apps/Hyper/install-32 [hunk 1]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+version=3.2.0
+
+install_packages https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${version}/hyper_${version}_armv7l.deb || exit 1
+
+if [[ "$(sudo cat /usr/share/applications/hyper.desktop)" =~ "Categories=TerminalEmulator;" ]]; then
+status "Fixing Hyper's Menu entry..."
+sudo cat /usr/share/applications/hyper.desktop | sed -e 's/Categories=TerminalEmulator;/Categories=Utility;/' | sudo tee /usr/share/applications/hyper.desktop &>/dev/null || error "Failed to fix desktop file!"
```
Comment (Botspot) [inline apps/Hyper/install-32:9]: This can be simplified with `sed -i`

File: apps/Hyper/install-64 [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/bash
+
+version=3.2.0
+
+install_packages https://github.com/Jai-JAP/hyper-arm-builds/releases/download/v${version}/hyper_${version}_arm64.deb || exit 1
+
+if [[ "$(sudo cat /usr/share/applications/hyper.desktop)" =~ "Categories=TerminalEmulator;" ]]; then
+status "Fixing Hyper's Menu entry..."
+sudo cat /usr/share/applications/hyper.desktop | sed -e 's/Categories=TerminalEmulator;/Categories=Utility;/' | sudo tee /usr/share/applications/hyper.desktop &>/dev/null || error "Failed to fix desktop file!"
```
Comment (Botspot) [inline apps/Hyper/install-64:9]: This can be simplified with `sed -i`

File: apps/Hyper/description [hunk 3]
```diff
@@ -0,0 +1,4 @@
+A modern electron terminal
```
Comment (Botspot) [inline apps/Hyper/description:1]: Try to be more descriptive and explain why it's better than alacritty, eDEX-UI, tabby, lxterminal, guake, etc.

**Comment by theofficialgman** [comment]
```
The goal of the project is to create a beautiful and extensible experience for command-line interface users, built on open web standards. In the beginning, our focus will be primarily around speed, stability and the development of the correct API for extension authors.
```
I fail to see how speed and web technologies can possibly go together.... but whatever, if people want more electron apps, have at it

**Comment by Jai-JAP** [comment]
This is also ready to merge.
Tested on both armhf and arm64

**Pull request merged**